json.c

```diff
@@ -1157,7 +1157,9 @@ static int json_write_pretty_get_object_size(const struct json_object_s* object,
 
   *size += depth * indent_size;
   *size += 1; // '}'
-  *size += newline_size; // need a newline next
+
+  if (0 != depth)
```

Comment:
if statement should have {} (one line if statements are bad chi for me!), and the indentation is wrong on the following line (looks like 4 spaces or a tab to me, where everything else is 2 spaces!). Good find though!